CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # CMakeList.txt : CMake project for XbSymbolSymbolDatabase, include source and define
 # project specific logic here.
 #
-cmake_minimum_required (VERSION 3.12)
+cmake_minimum_required (VERSION 3.10.2)
```

Comment:
Unfortunately, 3.12 is the minimal requirement due to `add_compile_definitions` function. Since it doesn't exist and certain CMake variant will make a fatal error by enforcing check what is used in here.

If there's a necessary to lower the minimal requirement, then an alternative solution is require before accept into upstream.

Reply:
Replaced `add_compile_definitions` with `target_compile_definitions` so that we're 3.10.2 compatible now.